### PR TITLE
Update TLS example

### PR DESCRIPTION
**Pull request by savitaashture**
# Changes

Modified TLS example as some issue introduced related to SA as part of this #884 
/cc @dibyom @khrm 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by khrm** [comment]
Closing https://github.com/tektoncd/triggers/pull/886 as both are same.

**Comment by dibyom** [comment]
LGTM once Kurram's comment is resolved!

File: examples/eventlistener-tls-connection/tls-eventlistener-interceptor.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ spec:
       interceptors:
         - github:
             secretRef:
-              secretName: github-secret
+              secretName: tls-secret
```
[the remark this answers is not in the document]
Comment (savitaashture) [reply examples/eventlistener-tls-connection/tls-eventlistener-interceptor.yaml]: done

**Review by khrm — APPROVED** [review]
/lgtm

**Pull request merged**